.env.baklava

```diff
@@ -118,3 +118,33 @@ LEADERBOARD_DOCKER_IMAGE_TAG="leaderboard-e84c5086e94193fdef430008bd44129e4151bb
 ATTESTATION_BOT_INITIAL_WAIT_SECONDS=600
 ATTESTATION_BOT_IN_BETWEEN_WAIT_SECONDS=600
 ATTESTATION_BOT_MAX_ATTESTATIONS=90
+
+# Number of gold-holding bots that vote for validator groups
+VOTING_BOTS=100
+# 10,000 CG
+VOTING_BOT_BALANCE=10000000000000000000000
```

Comment:
With only 10k Celo Gold, will the bots have enough impact?

Reply:
This is what @timmoreton suggested. At this amount, each bot has slightly less than each validator. (Each validator gets 12k, and each group gets 12k).

Tim - what was your rationale for that amount? Is this still in line with the level of impact we want each bot to have?

I think it's fine to have this as a starting point. We can adjust if we want more impact - either by adding more bots, or giving each bot more gold. Given the current setup function, adding more bot accounts is a bit easier than increasing the locked gold for existing ones.

Reply:
Ah, right.  I think I forgot that essentially all the validator groups would be tied.